### DISCUSSION TITLE
Add Search Filter on Gastro Orders history

### DISCUSSION THREAD

**Discussion by mbugikp** [Blair2004/NexoPOS#810] [Ideas]
**Describe the bug**
We found that when you want to narrow down on Gastro orders history you can only search by date no other option, we therefore propose we add a search filter on Gastro orders history with options for author / waiter, order code, amount etc

**To Reproduce**
1. Open POS
2. Go to tables
3. Click table management
4. Choose table
5. Click table
![Search Filter on Gastro Orders History](https://user-images.githubusercontent.com/13999977/161089234-34f9339b-675e-4d0b-82f3-4ba9ac67635d.png)

**Expected behavior**
Replace settings with Search Filter

**Screenshots**

**Environment:**
 - NexoPOS 4.6.16
 - Premium Extensions: Gastro 4.6.6

**Comment by mbugikp** [comment]
Thank you @Blair2004 the search filter will be a great feature, helps if you have many uncleared orders from different waiters on different days when the cashier wants to clear the orders he will start with orders of a particular waiter on a particular day, with the search filter it will be faster because he will search for only orders of that waiter on that particular day then click pay button to clear them.

**Reply by Blair2004** [reply]
That will be a bit complex, but we have that saved on the side.
Thank you @mbugikp
